pages/whats-new/research.md

```diff
@@ -6,11 +6,10 @@ redirect_from:
 layout: styleguide
 title: Research
 category: What’s new
-lead: User research will be a core aspect of the U.S. Web Design System project as it’s our main source of feedback and inspiration for future product development.
+lead: User research is a core aspect of the U.S. Web Design System as it’s our main source of feedback and inspiration for future product development.
 ---
-We plan to use a combination of research methods, quantitative research like collecting web analytics to see in how frequently the Design System is used around the federal government, and qualitative research like remote and in-person observational studies to see whether the Design System is making government sites easier for people to use.
 
-We’ll use this research to see how well the Design System is working and what needs to be improved. We can also provide user research services to teams that are using the Design System. These will be priced to allow teams around the federal government to engage with us and improve their product for their users. This benefits the Design System as well because we’ll get feedback on where they need to be improved and extended.
+We plan to use a combination of research methods, quantitative research like collecting web analytics to see in how frequently the Design System is used around the federal government, and qualitative research like remote observational studies to see whether the Design System is making government sites easier for people to use. We’ll use this research to see how well the Design System is working and what needs to be improved. 
```

Comment:
Should the `in` be removed from `to see in how`?

Reply:
Fixed in 7678f0c.